src/ui/public/dashboard_panel_actions/dashboard_context_menu_panel.ts

```diff
@@ -47,7 +48,7 @@ export class DashboardContextMenuPanel {
   /**
    * Optional, could be composed of actions instead of content.
    */
-  public getContent(panelActionAPI: PanelActionAPI): JSX.Element | Element | undefined {
+  public getContent(panelActionAPI: PanelActionAPI): ReactElement<any> | HTMLElement | undefined {
```

Comment:
Curious to know why this change was necessary.

Reply:
That is what the icon type that is so I followed suit.  To be honest, I'm not 100% sure what these sure be but I went with the result of this conversation in EUI: https://github.com/elastic/eui/issues/985#issuecomment-404630178

I think Element was giving me an issue but HTMLElement was not as something that could be passed into EUI.  See this too for context: https://github.com/elastic/eui/issues/1098

We just need to define a type of element that plugins can use that can be react, but doesn't have to be.  Which I guess is still what it was originally, but I *think* these types are better.  @chandlerprall can maybe weigh in too as I think we kinda settled on these two types over the original two.